src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
```

Comment:
Remember to add empty lines between multiline sibling blocks of HTML. This helps to add some 'air' and simplify reading. So, there should be an empty line between each 'div' with the class 'stars'. Refer to the checklist for examples.